### PR TITLE
Add OWNERS files

### PR DESCRIPTION
**Pull request by evankanderson**

### CODE REVIEW

**Comment by n3wscott** [comment]
looks like grantr is gone now

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+  - devguyio
+  - grantr
+  - lionelvillard
+  - vaikas
+  eventing-writers:
+  - akashrv
+  - aliok
+  - antoineco
+  - devguyio
+  - grantr
```
[the remark this answers is not in the document]
Comment (evankanderson) [reply OWNERS_ALIASES:67]: Thanks, will do that now. 6 interviews in 5 days does not leave as much time for other things as you might expect. 😁

**Review by dprotaso — APPROVED** [review]

**Comment by dprotaso** [comment]
ok - I think a TOC member needs to deal with this

**Pull request merged**